modules/cas_cache/main.c

```diff
@@ -83,7 +83,9 @@ int static cas_find_symbol(void *data, const char *namebuf,
 
 int static cas_casdisk_lookup_funtions(void)
 {
+#ifdef MODULE_MUTEX_SUPPORTED
```

Comment:
We also need to change line 49 to
```c
#if defined(SYMBOL_LOOKUP_SUPPORTED) && defined(MODULE_MUTEX_SUPPORTED)
```
so that only if both are supported, the dynamic lookup is performed.